src/components/Button.vue

```diff
@@ -15,11 +15,11 @@
     @click="$emit('click')"
     :target="target"
     @mousedown="pressDown"
-    :style="rippleStyle"
+    :style="[rippleStyle, buttonStyles]"
```

Comment:
What is inside the buttonStyles, this might clash with the ripple effect on the buttons,